libs/core/kiln_ai/adapters/ml_model_list.py

```diff
@@ -1225,6 +1225,12 @@ class KilnModel(BaseModel):
                 supports_structured_output=False,
                 supports_data_gen=False,
             ),
+            KilnModelProvider(
+                name=ModelProviderName.docker_model_runner,
+                model_id="ai/llama3.2:3B-Q4_K_M",
+                structured_output_mode=StructuredOutputMode.json_schema,
+                supports_data_gen=False,
+            ),
```

Comment:
It would be good to add the others in this PR. The usability of DMR goes way up with them.

Reply:
Added in https://github.com/Kiln-AI/Kiln/pull/473/commits/6fc2711bdfe0c5665e52b3ea44699614b5f897da